fastplotlib/graphics/text.py

```diff
@@ -16,6 +16,26 @@ def __init__(
             outline_thickness=0,
             name: str = None,
     ):
+        """
+        Create a text Graphic
+
```

Comment:
just keep this as is, text is very primitive so it's not useful to document this yet.